src/ontology/metadata/medgen.yml

```diff
@@ -0,0 +1,11 @@
+id: MEDGEN
```

Comment:
## Temporary `metadata/SOURCE.yml` changes @matentzn 
@matentzn Just need you to look this over.
<details><summary>Details</summary>
<p>

These are temporary measures until the EPM refactor is done:
- #146

These changes are prompted by recent edits I made to `prefixes.csv`. I want these files to be consistent with that.

### MedGen
I created a new `metadata/medgen.yml` file, but I didn't create it the standard way (couldn't remember how to, and I didn't think it was worth the time). I just wanted to make sure that since it is a source, and since I found multiple URI stems for it, that information should be present when we do the refactor so that it is not lost.

@matentzn I just want to make sure that creating this file as such won't screw anything up.

### ICD10WHO
The file was actually inconsistent:
```yml
prefix_map:
  ICD10WHO: http://apps.who.int/classifications/icd10/browse/2010/en#/
base_prefix_map:
  ICD10WHO: https://icd.who.int/browse10/2019/en#/
```

I've updated it to use 2 different prefix maps for these 2 URIs, and be consistent with `prefixes.csv`

@matentzn Just want to make sure this doesn't screw anything up.

### OMIM
@matentzn I don't think this change will screw anything up here.

I saw that there was a second way that OMIMPS URIs could be formulated, so I added a new prefix map for that.

</p>
</details>

Reply:
why is all this needed for this specific PR? medgen is not even a real source in the strict sense..

Reply:
I attempt to explain why in my comment above. It's not only about MedGen; that was just closely coupled with these other (I think necessary) changes. Technically I can (and will now) remove the MedGen part into a different PR, but I will leave these other changes for your review (see _details_ in my comment).